README.md

```diff
@@ -1,2 +1,3 @@
 # JavaScriptLearning
 Contains practice codes I did while learning JavaScript.
+ 	Day 1: Learning JavaScript Fundamentals.
```

Comment:
Please add some more files, only then will I have something to review.

Reply:
Ok, I was just testing pull request option. I will add some files for next pull request.